extensions/interactions/base_test.py

```diff
@@ -355,80 +355,75 @@ def test_default_interactions_are_valid(self):
         all_interaction_ids = (
             interaction_registry.Registry.get_all_interaction_ids())
         for interaction_id in all_interaction_ids:
+            # TODO(#10212):Ignoring the RatioExpressionInput for now as
+            # there's no learner's flow for the interaction yet.
+            # Once added this needs to be removed.
+
             # Check that the interaction id is valid.
             self.assertTrue(self._is_camel_cased(interaction_id))
             hyphenated_interaction_id = (
                 utils.camelcase_to_hyphenated(interaction_id))
-
             # Check that the interaction directory exists.
             interaction_dir = os.path.join(
                 feconf.INTERACTIONS_DIR, interaction_id)
             self.assertTrue(os.path.isdir(interaction_dir))
-
```

Comment:
Why is this removed?

Reply:
Also, a lot of unneeded changes in this file.

Reply:
updated